update_scripts/pyiron_base_0.3_to_0.4.py

```diff
@@ -0,0 +1,15 @@
+"""
+pyiron_base<=0.3.9 has a bug that writes all arrays with dtype=object even
```

Comment:
```suggestion
pyiron_base<=0.3.10 has a bug that writes all arrays with dtype=object even
```
I just looked it up and 0.3.10 does not yet have the writing fixed - only the master becoming 0.4.0 has the fix.